src/main/java/org/testng/annotations/Ignore.java

```diff
@@ -0,0 +1,24 @@
+package org.testng.annotations;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PACKAGE;
+import static java.lang.annotation.ElementType.TYPE;
+
+/**
+ * Alternative of @Test(enable=false)
+ *
+ * Notice that @Ignore on a class will disable all test methods of the class.
+ *
+ * Ignoring a class will ignore tests from child classes too.
+ *
+ * Ignoring a package will ignore all tests in the package and its sub-packages
```

Comment:
It still doesn't tell me how exactly do I add the intent of ignoring tests at a package level. 

Maybe we can add https://github.com/cbeust/testng/pull/861/files#diff-74241240e3f2f62c69f7ef5bb21867b9 contents to the java docs.

Reply:
ok. Annotations on package are not easy :)

Reply:
done